avaframe/in3Utils/cfgUtils.py

```diff
@@ -202,7 +202,19 @@ def compareConfig(iniFile, modName, compare, modInfo=False, toPrint=True):
         for section in locCfg.sections():
             if defCfg.has_section(section):
                 for key in locCfg.items(section):
-                    log.warning('Additional Key [\'%s\'] in section [\'%s\'] is ignored.' % (key[0], section))
+                    # an exception is made for thickness values that are added for the features of a releaseScenario,
+                    # entrainment Scenario or secondar. release scenario
+                    # these are added to the configuration and also to the modDict if variation is applied
+                    validItems = ['entrainmentScenario', 'DEM', 'secondaryReleaseScenario']
+                    if 'relTh' in key[0] or 'entTh' in key[0] or 'secondaryRelTh' in key[0] or key[0] in validItems:
```

Comment:
rewrite to if `key[0] in ...` ?

Reply:
here again it should also find e.g. relTh0 so we need to check if 'relTh' in key[0]

Reply:
Same as before:
`searchKeys = ['relTh','entTh'....]
if any(k in key[0] for k in searchKeys)
`?

Reply:
validItems = ['entrainmentScenario', 'DEM', 'secondaryReleaseScenario']
searchItems = ['relTh', 'entTh', 'secondaryRelTh']
if any(s in key[0] for s in searchItems) or key[0] in validItems: